nss-ivory-trolls/BangazonSite
# Ab ticket4 2

# Description

**Given** the user is on any page of the Bangazon site
**When** the user performs a gesture on a product category name
**Then** a list of all products in that category will be displayed, with title, quantity, and price/unit


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

```
git fetch --a
git checkout ab-ticket4-2
```

1. Run the program
2. Click the "Products" nav item
3. Confirm that the product types are all links that direct to a view of all products under that product type


# Checklist:
